t/12_sles4sap_publicccloud.t

```diff
@@ -0,0 +1,31 @@
+use strict;
+use warnings;
+use testapi;
+use Test::MockModule;
+use Test::Exception;
+use Test::More;
+use sles4sap_publiccloud;
+
+
+subtest "Run 'setup_sbd_delay' with different values" => sub {
+    my $self = sles4sap_publiccloud->new();
+    my $sles4sap_publiccloud = Test::MockModule->new('sles4sap_publiccloud', no_auto => 1);
+    $sles4sap_publiccloud->redefine(record_info => sub { return; });
+    $sles4sap_publiccloud->redefine(cloud_file_content_replace => sub { return; });
+    $sles4sap_publiccloud->redefine(croak => sub { die; });
+
+    my @passing_values = qw(1 yes no 100 100s);
+    my @failok_values = qw(aasd 100asd);
+
+    for my $value (@failok_values) {
+        set_var('HA_SBD_START_DELAY', $value);
+        dies_ok { $self->setup_sbd_delay() } "Test expected failing 'HA_SBD_START_DELAY' value: $value";
+    }
+
+    for my $value (@passing_values) {
+        set_var('HA_SBD_START_DELAY', $value);
+        ok($self->setup_sbd_delay(), "Test 'HA_SBD_START_DELAY' passing values: $value");
+    }
```

Comment:
Please call `set_var('HA_SBD_START_DELAY', undef);` at the end

Reply:
Yes, good catch. Thanks.